go.mod

```diff
@@ -4,24 +4,25 @@ go 1.15
 
 require (
 	github.com/Azure/go-autorest/autorest v0.11.6 // indirect
-	github.com/go-logr/logr v0.3.0
+	github.com/cloudflare/cfssl v1.5.0
+	github.com/go-logr/logr v0.4.0
 	github.com/jetstack/cert-manager v0.0.0-00010101000000-000000000000
 	github.com/kr/pretty v0.2.1 // indirect
-	github.com/open-cluster-management/api v0.0.0-20201007180356-41d07eee4294
+	github.com/open-cluster-management/addon-framework v0.0.0-20210414095446-30a5d245b8c7
+	github.com/open-cluster-management/api v0.0.0-20210409125704-06f2aec1a73f
 	github.com/open-cluster-management/multicloud-operators-placementrule v0.0.0-20210325184301-dd3e27fc2978
 	github.com/open-cluster-management/observatorium-operator v0.0.0-20210329030346-6ff35cf8bcd2
 	github.com/openshift/api v3.9.1-0.20190924102528-32369d4db2ad+incompatible
-	github.com/openshift/client-go v0.0.0-20201020082437-7737f16e53fc
-	golang.org/x/tools v0.0.0-20201014231627-1610a49f37af // indirect
-	gopkg.in/yaml.v2 v2.3.0
+	github.com/openshift/client-go v0.0.0-20201214125552-e615e336eb49
+	gopkg.in/yaml.v2 v2.4.0
 	k8s.io/api v0.20.5
 	k8s.io/apiextensions-apiserver v0.20.1
 	k8s.io/apimachinery v0.20.5
 	k8s.io/client-go v12.0.0+incompatible
 	k8s.io/kubectl v0.20.4
 	k8s.io/utils v0.0.0-20201110183641-67b214c5f920
 	sigs.k8s.io/controller-runtime v0.8.0
-	sigs.k8s.io/kube-storage-version-migrator v0.0.3-0.20210302135122-481bd04dbc78
+	sigs.k8s.io/kube-storage-version-migrator v0.0.3
```

Comment:
@morvencao Does it work?

Reply:
It should be working, as long as we have correct version in `replace` stanza.